tools/cli/lib/parseargs.c

```diff
@@ -80,6 +83,17 @@ static struct option pstOptions[] =
     {"disableexcludes", no_argument, &_opt.nDisableExcludes, 1}, //--disableexcludes
     {"downloadonly",  no_argument, &_opt.nDownloadOnly, 1}, //--downloadonly
     {"downloaddir",   required_argument, 0, 0},            //--downloaddir
+    // reposync options
+    {"arch",          required_argument, 0, 0},
```

Comment:
These new commands and options needs to be added to help.c

Reply:
Good point. Fixed.